include/wx/msw/apptrait.h

```diff
@@ -87,6 +87,31 @@ class WXDLLIMPEXP_CORE wxGUIAppTraits : public wxGUIAppTraitsBase
     virtual bool WriteToStderr(const wxString& WXUNUSED(text)) { return false; }
 };
 
+#elif defined(__WXQT__)
```

Comment:
This header starts getting really messy. I wonder if it could be better to have port-specific (rather than platform-specific) traits classes, at least for QT?